engine/src/operators/OrderBy.cpp

```diff
@@ -321,9 +321,187 @@ void process_sort(blazing_frame & input, std::string query_part, Context * query
 	}
 }
 
+
+
+}  // namespace operators
+}  // namespace ral
+
+namespace ral {
+namespace operators {
+namespace experimental {
+
+	using Context = blazingdb::manager::experimental::Context;
+	using Node = blazingdb::transport::experimental::Node;
+	using namespace ral::distribution::experimental;
+
+std::unique_ptr<ral::frame::BlazingTable> process_sort(const ral::frame::BlazingTableView & table, std::string query_part, Context & context) {
```

Comment:
Drive by review, don't pass strings by value :) 
```suggestion
std::unique_ptr<ral::frame::BlazingTable> process_sort(const ral::frame::BlazingTableView & table, std::string const& query_part, Context & context) {
```